pkg/timezone/timezone_unix.go

```diff
@@ -0,0 +1,25 @@
+//go:build darwin || linux
+
+package timezone
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+)
+
+// Timezone returns the local timezone if found. If not, an error is returned.
+func Timezone() (string, error) {
+	tz, err := os.Readlink("/etc/localtime")
+	if err != nil {
+		return "", fmt.Errorf("failed to read symlink for localtime")
+	}
+
+	idx := strings.Index(tz, "zoneinfo/")
+	if idx == -1 {
+		return "", fmt.Errorf("'zoneinfo' not found at path for %s", runtime.GOOS)
+	}
+
+	return tz[idx+9:], nil
```

Comment:
This may result in `panic: runtime error: slice bounds out of range` so let's check the length of `tz` beforehand.